VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift

```diff
@@ -220,30 +220,15 @@ class SendCryptoViewModel: ObservableObject {
                     await BalanceService.shared.updateBalance(for: tx.coin)
                     
                     var gas = BigInt.zero
-                    if percentage == 100 {
-                        // For max sends, fetch actual gas estimation instead of using default fee
-                        let polkadotSpecific = try await blockchainService.fetchSpecific(tx: tx)
-                        gas = polkadotSpecific.gas
-                    }
+                    let dot = try await blockchainService.fetchSpecific(tx: tx)
+                    gas = dot.gas
                     
                     tx.amount = "\(tx.coin.getMaxValue(gas).formatToDecimal(digits: tx.coin.decimals))"
                     setPercentageAmount(tx: tx, for: percentage)
                     
                     convertToFiat(newValue: tx.amount, tx: tx, setMaxValue: tx.sendMaxAmount)
                 } catch {
-                    // Fallback to default fee if gas estimation fails
-                    await BalanceService.shared.updateBalance(for: tx.coin)
-                    
-                    var gas = BigInt.zero
-                    if percentage == 100 {
-                        gas = tx.coin.feeDefault.toBigInt()
-                    }
-                    
-                    tx.amount = "\(tx.coin.getMaxValue(gas).formatToDecimal(digits: tx.coin.decimals))"
-                    setPercentageAmount(tx: tx, for: percentage)
-                    
-                    convertToFiat(newValue: tx.amount, tx: tx, setMaxValue: tx.sendMaxAmount)
-                    print("Failed to get DOT gas estimation, using default fee: \(error.localizedDescription)")
+                    throw error
```

Comment:
This might cause the app to crash , because `setMaxValues` doesn't have any other try/catch,